skeleton/techdoc/docs/rhoai.md

```diff
@@ -12,37 +12,37 @@ This document will outline how you can run build and run your sample application
 
 ## Running the Sample
 
-1) While on Console, navigate to the `OpenShift AI` by clicking the square "apps" icon on the top-right corner (next to the notifications icon). `Openshift AI` will be listed in the dropdown list
+1) On the Console, click the square "apps" icon on the upper-right corner (next to the notifications icon). `Openshift AI` is listed in the drop-down list.
 
 ![image](./images/access-openshift-ai.png)
 
-2) Go to the `Data Science Projects` section and access your application's project named `{{ values.namespace }}`
+2) Go to the `Data Science Projects` section and access your application's project named `{{ values.namespace }}`.
 
 ![image](./images/data-science-projects.png)
 
-3) Access the `workbench` named `{{ values.name }}-notebook`
+3) Access the `workbench` named `{{ values.name }}-notebook`.
 
 ![image](./images/access-workbench.png)
 
-4) Go to `File->Open` and select `Terminal`
+4) Go to `File->Open` and select `Terminal`.
 
 ![image](./images/open-terminal.png)
 
-5) In the terminal, run `cd ${{ values.name }}` to navigate to your sample app's directory
+5) In the terminal, run `cd ${{ values.name }}` to navigate to your sample app's directory.
 
-6) Run `pip install --upgrade -r requirements.txt` to install the dependencies for your application
+6) Run `pip install --upgrade -r requirements.txt` to install the dependencies for your application.
 
 7) Run `${{ values.appRunCommand }}` to run the sample in the workbench.
 
 ## Accessing the Sample
 
-With the sample app now running, the following steps will allow you to access the sample app in your browser:
+With the sample app now running, complete the following steps to access the sample app in your browser:
 
-1) Navigate back to the OpenShift AI dashboard, and find the name of your workbench.
+1) Go to the OpenShift AI dashboard, and find the name of your workbench.
 ![image](./images/workbench-name.png)
 
-2) In a terminal window on your machine, run `oc get pods -l app=<workbench-name>`. This will retrieve the name of the pod where the workbench is running.
+1) In a terminal window on your machine, run `oc get pods -l app=<workbench-name>`. This retrieves the name of the pod where the workbench is running.
```

Comment:
Should this be numbered as 1 or 2? When I made a change to this doc it incorrectly swapped it to a 1 without my doing so just thought I'd ask

Reply:
updated